tough/src/error.rs

```diff
@@ -440,6 +440,10 @@ pub enum Error {
 
     #[snafu(display("Role missing from snapshot meta: {}", name))]
     RoleNotInMeta { name: String },
+
+    /// Root creates an unloadable repo
+    #[snafu(display("Unstable root, less keys then threshold for: {}", role))]
```

Comment:
Since we know how many keys we have (and the threshold), let's include it in the error message.  Suggestion:

`"Unstable root; found {} keys for role {}, threshold is {}"`